### PR TITLE
chore: update release number in the README.md file

### PR DESCRIPTION
**Pull request by rjuliano**
*Issue #, if available:*

*Description of changes:*
Update the version in the README.md file when we generate a release.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by richardmcclellan — COMMENTED** [review]
This is great.  We also usually update `rxbindings/README.md` - could we add that in?

I wonder if there's a way we could make this logic generic enough to work for all 3 of our repos, though that may be an over optimization.

**Comment by rjuliano** [comment]
> This is great. We also usually update `rxbindings/README.md` - could we add that in?
Yup...it was easy enough to do that.

> I wonder if there's a way we could make this logic generic enough to work for all 3 of our repos, though that may be an over optimization.
I changed the `update_docs` function to take two additional parameters: `file_path` and `maven_group_name`. `file_path` makes it so we can pass any file we may want to update. `maven_group_name` will come in as a command line parameters (for this repo, it's `com.amplifyframework`)

File: scripts/fastlane/Fastfile [hunk 1]
```diff
@@ -26,6 +28,7 @@ platform :android do |options|
   lane :create_next_release_pr do |options|
     # This will be used in the commit and PR title
     product_name = options[:product_name]
+    maven_group_name = options[:maven_group_name]
```
Comment (richardmcclellan) [inline scripts/fastlane/Fastfile:31]: Looks like the package name is defined [in the Appfile, here](https://github.com/aws-amplify/amplify-android/blob/main/scripts/fastlane/Appfile#L2), so another option might be to grab the value from there using: 

```
maven_group_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name)
```

Ref: https://docs.fastlane.tools/advanced/Appfile/

I'm not sure if that's better or not, but it's another option.

Comment (rjuliano) [reply scripts/fastlane/Fastfile:31]: Let me try it. It would be one less thing to pass via the command line.

**Review by richardmcclellan — APPROVED** [review]

File: scripts/fastlane/Fastfile (hunk 1 above)
Comment (rjuliano) [reply scripts/fastlane/Fastfile]: That worked! Just updated the PR

**Pull request merged**